<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerKarteaConfigEditView</class>
 <widget class="QDialog" name="PlayerKarteaConfigEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plataforma T-TEA - Configuração Kartea</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/resources.qrc">
    <normaloff>:/icons/system/appicon</normaloff>:/icons/system/appicon</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="lay_game_setting">
      <attribute name="title">
       <string>Configuração de Jogo</string>
      </attribute>
      <layout class="QVBoxLayout" name="tab_game_setting">
       <item>
        <layout class="QFormLayout" name="frm_game_setting">
         <property name="labelAlignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_player">
           <property name="text">
            <string>Jogador:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbx_player">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_session">
           <property name="text">
            <string>Sessão:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lbl_session_value">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_current_phase">
           <property name="text">
            <string>Fase Atual:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbx_current_phase"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_current_level">
           <property name="text">
            <string>Nível Atual:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cbx_current_level"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl_level_time">
           <property name="text">
            <string>Tempo de Nível:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spn_level_time">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs_game_setting">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lay_visual_resources">
      <attribute name="title">
       <string>Recurso Visual</string>
      </attribute>
      <layout class="QVBoxLayout" name="visual_resources_layout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_visual_current_type">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Selecione um recurso visual para visualizar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="lbl_visual_preview">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>280</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>350</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid #666666; background-color: #ffffff; border-radius: 10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_preview">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="visual_form">
         <property name="labelAlignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_vehicle_image">
           <property name="text">
            <string>Imagem do Veículo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbx_vehicle_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_environment_image">
           <property name="text">
            <string>Imagem do Ambiente:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbx_environment_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_target_image">
           <property name="text">
            <string>Imagem do Alvo:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbx_target_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl_obstracle_image">
           <property name="text">
            <string>Imagem do Obstáculo:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cbx_obstacle_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs_visual_resource">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lay_visual_feedback">
      <attribute name="title">
       <string>Feedback Visual</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_visual_feedback">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="lbl_current_feedback_type">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Selecione uma imagem de feedback para visualizar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="lbl_feedback_preview">
         <property name="minimumSize">
          <size>
           <width>280</width>
           <height>280</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>350</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid #666666; background-color: #ffffff; border-radius: 10px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="vs_preview">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QFormLayout" name="frm_visual_feedback">
         <property name="labelAlignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_positive_feedback_image">
           <property name="text">
            <string>Imagem de Feedback Positivo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbx_positive_feedback_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_neutral_feedback_image">
           <property name="text">
            <string>Imagem de Feedback Neutro:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbx_neutral_feedback_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_negative_feedback_image">
           <property name="text">
            <string>Imagem de Feedback Negativo:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbx_negative_feedback_image">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs_visual_feedback">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lay_sound_feedback">
      <attribute name="title">
       <string>Feedback Sonoro</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QFormLayout" name="frm_sound_feedback">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_sound_feedback_positive">
           <property name="text">
            <string>Som de Feedback Positivo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="lay_positive_feedback_sound">
           <item>
            <widget class="QComboBox" name="cbx_positive_feedback_sound">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_positive_sound">
             <property name="text">
              <string>Play</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../resources/resources.qrc">
               <normaloff>:/icons/ui/buttons/playicon</normaloff>:/icons/ui/buttons/playicon</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_sound_feedback_neutral">
           <property name="text">
            <string>Som de Feedback Neutro:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="lay_neutral_feedback_sound">
           <item>
            <widget class="QComboBox" name="cbx_neutral_feedback_sound">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_neutral_sound">
             <property name="text">
              <string>Play</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../resources/resources.qrc">
               <normaloff>:/icons/ui/buttons/playicon</normaloff>:/icons/ui/buttons/playicon</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_sound_feedback_negative">
           <property name="text">
            <string>Som de Feedback Negativo:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="lay_negative_feedback_sound">
           <item>
            <widget class="QComboBox" name="cbx_negative_feedback_sound">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_negative_sound">
             <property name="text">
              <string>Play</string>
             </property>
             <property name="icon">
              <iconset resource="../../../../resources/resources.qrc">
               <normaloff>:/icons/ui/buttons/playicon</normaloff>:/icons/ui/buttons/playicon</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs_sound_feedback">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lay_interface_setting">
      <attribute name="title">
       <string>Configuração de Interface</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QFormLayout" name="frm_interface_setting">
         <item row="0" column="0">
          <widget class="QLabel" name="lbl_palette">
           <property name="text">
            <string>Paleta:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spn_palette">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>999999</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl_hud">
           <property name="text">
            <string>HUD:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chk_hud"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl_sound">
           <property name="text">
            <string>Som:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="chk_sound"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vs_interface_setting">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lay_button">
     <item>
      <spacer name="hs_button">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ok">
       <property name="toolTip">
        <string>Gravar operação corrente</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/resources.qrc">
         <normaloff>:/icons/ui/buttons/okicon</normaloff>:/icons/ui/buttons/okicon</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="toolTip">
        <string>Cancelar operação corrente</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/resources.qrc">
         <normaloff>:/icons/ui/buttons/cancelicon</normaloff>:/icons/ui/buttons/cancelicon</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
